rmgpy/kinetics/surface.pyx

```diff
@@ -426,8 +512,67 @@ cdef class SurfaceArrhenius(Arrhenius):
         A helper function used when pickling a SurfaceArrhenius object.
         """
         return (SurfaceArrhenius, (self.A, self.n, self.Ea, self.T0, self.Tmin, self.Tmax, self.Pmin, self.Pmax,
-                                   self.uncertainty, self.comment))
+                                   self.coverage_dependence, self.uncertainty, self.comment))
+
+    cpdef double get_rate_coefficient(self, double T, double P=0.0) except -1:
```

Comment:
why is this now needed but wasn't before? is it different from the Arrhenius class we inherit from?

Reply:
It isn't. It was leftover from something I did a long time ago. I will remove it!